common/org.testeditor.dsl.common.testing/build.properties

```diff
@@ -2,4 +2,5 @@ source.. = src/,\
            xtend-gen/
 output.. = bin/
 bin.includes = META-INF/,\
-               .
+               .,\
+               lib/mockito-all-1.10.19.jar
```

Comment:
:+1:
Still, in the long run we should find a better solution for adding third-party libs to the target platform. Mockito is available in Orbit, but does not work properly due to signing issues - maybe Mockito 2.0 will provide an alternative once released (I think it was planned to use ByteBuddy instead of cglib).